packages/frontend-api/src/Model/Resolver/Advert/AdvertsQuery.php

```diff
@@ -44,7 +48,12 @@ public function advertsQuery(?string $positionName = null, ?string $categoryUuid
         $category = null;
 
         if ($categoryUuid !== null) {
-            $category = $this->categoryFacade->getByUuid($categoryUuid);
+            try {
+                $category = $this->categoryFacade->getByUuid($categoryUuid);
+            } catch (CategoryNotFoundException $exception) {
+                $seoMixCategory = $this->readyCategorySeoMixFacade->getByUuid($categoryUuid);
```

Comment:
Two points:
1. $categoryUuid should be renamed to `CategoryOrSeoCategoryUuid` (and in API?)
2. `ReadyCategorySeoMixFacade` is in project-base, It means packages/frontend-api depends on project-base